tests/test_step_simulation_ops.py

```diff
@@ -1,5 +1,7 @@
 from unittest import TestCase
```

Comment:
here's probably the root cause: our testsuite for step_simulation_ops lacks good coverage.

Reply:
Yeah that's a good point. It would be nice to build out that coverage